cnv/cnv_virtual_machines/cnv_vm_networking/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -16,6 +16,14 @@ include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
 
 == Creating a NetworkAttachmentDefinition
 
+.Prerequisites
+
+* A Linux bridge must be xref:../../../cnv/cnv_node_network/cnv-updating-node-network.adoc[configured and attached] to the correct
+Network Interface Card (NIC) on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
```

Comment:
I think we need a more specific language here:

If you use VLANs -> if you intend to connect multiple VLANs via this bridge

Reply:
Actually, we can drop this mention. nmstate enables vlan filtering by default.